src/translations/strings.yaml

```
@@ -228,7 +228,7 @@ notSignedInGoogle:
 # IAP Android edge case 2
 multiFxaAccountError:
   fxaAccountErrorHeader: Problem confirming subscription
-  fxaAccountErrorText: Your subscription is linked to another Firefox Account. You may need to sign in with a different email address.
+  fxaAccountErrorText2: Your subscription is linked to another Mozilla account. You may need to sign in with a different email address.
```

```suggestion
  mozAccountErrorText: Your subscription is linked to another Mozilla account. You may need to sign in with a different email address.
```

Maybe aboid fxa in the ID at this point?